.github/workflows/rpc-compat.yml

```diff
@@ -0,0 +1,34 @@
+name: rpc-compat
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  check-rpc-for-breaking-changes:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup sentry env
+        uses: ./.github/actions/setup-sentry
+        id: setup
+      - name: Dump RPC schema for new version
+        id: newschema
+        run: |
+          mkdir schemas
+          sentry rpcschema --partial > schemas/new_schema.json
+      - name: Dump RPC schema for old version
+        id: oldschema
+        run: |
+          git checkout `git merge-base origin/${{ github.base_ref }} origin/${{ github.head_ref }}`
+          sentry rpcschema --partial > schemas/old_schema.json
```

Comment:
We expect this step to fail in *this* PR because the `sentry rpcschema` command doesn't exist in master yet (and won't until this branch is merged). We can safely ignore it.

If I'm visualizing the Git tree correctly, this problem shouldn't recur for any open branches at the time we merge this, because the job itself won't exist until the branch is synced from master.